examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -60,7 +60,7 @@
    "id": "d8642b00",
```

Comment:
<p>Is the number of the heading correct?  I would number this as 3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>